### ISSUE TITLE
documentation bugs

### ISSUE THREAD

**Issue by aburrell** [rstoneback/pysat#153]
There are examples in the docs that are wrong:
https://pysat.readthedocs.io/en/latest/examples.html?highlight=restrictMLAT

```
# perform seasonal average
ivm.bounds(startDate, stopDate)
```

and it should be

```
# perform seasonal average
ivm.bounds = (startDate, stopDate)
```

**Comment by jklenzing** [comment]
Going to go through the example code and make sure it all does what it's supposed to.

**Comment by jklenzing** [comment]
While I'm at it, I could update the sample documentation images with the uniform sequential colormaps.  Thoughts @rstoneback?

**Comment by rstoneback** [comment]
Sure. If you have a better display idea then give it a shot. Thanks for picking this up as well. This one completely got past me. Didn’t even notice it come in.

On Feb 21, 2019, at 9:24 PM, Jeff Klenzing <notifications@github.com<mailto:notifications@github.com>> wrote:


While I'm at it, I could update the sample documentation images with the uniform sequential colormaps. Thoughts @rstoneback<https://github.com/rstoneback>?

**Comment by jklenzing** [comment]
Largely thinking of improving accessibility, plus matching what a new user would see now that `matplotlib` has swapped the default colormap.

**Comment by rstoneback** [comment]
Sounds good.

A bigger project, but I believe there is a way to get the documentation to actually run the example code. Would help guard against the kinds of errors Angeline found.


On Feb 21, 2019, at 9:36 PM, Jeff Klenzing <notifications@github.com<mailto:notifications@github.com>> wrote:


Largely thinking of improving accessibility, plus matching what a new user would see now that matplotlib has swapped the default colormap.

**Comment by jklenzing** [comment]
The last two examples don't include the start and stop dates.  Any suggestions?

**Comment by jklenzing** [comment]
I've updated the first two graphs in the documentation branch.  There is not enough information in the second two blocks of code to generate these graphs.  In particular, the multiple functions to add are not found.  It works as an example of what could be done, but as a new user, I would find this frustrating.

**Comment by jklenzing** [comment]
Aha!  Found the demo files.  Ideally, we should link these so that when one is updated, the other one is too.

**Comment by rstoneback** [comment]
Fair enough.

I think I put the full code in the demo area of the repo.

On Feb 21, 2019, at 10:16 PM, Jeff Klenzing <notifications@github.com<mailto:notifications@github.com>> wrote:


I've updated the first two graphs in the documentation branch. There is not enough information in the second two blocks of code to generate these graphs. In particular, the multiple functions to add are not found. It works as an example of what could be done, but as a new user, I would find this frustrating.

**Comment by jklenzing** [comment]
OK, while I'm in here, I'm going to play with style a little.  Probably will be some adjustments.  Check out the first example in the documentation branch here:
https://github.com/rstoneback/pysat/blob/documentation/docs/examples.rst

**Comment by jklenzing** [comment]
Closed with merge of #175.
